### PR TITLE
Add new queryString argument to initializeEdit function.

### PR DESCRIPTION
**Pull request by corivera**
Part of the set of changes needed to support using generic SQL queries to retrieve EditData rows.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -494,9 +494,9 @@ export class QueryFeature extends SqlOpsFeature<undefined> {
 			);
 		};
 
-		let initializeEdit = (ownerUri: string, schemaName: string, objectName: string, objectType: string, LimitResults: number): Thenable<void> => {
+		let initializeEdit = (ownerUri: string, schemaName: string, objectName: string, objectType: string, LimitResults: number, queryString: string): Thenable<void> => {
```
Comment (anthonydresser) [inline src/main.ts]: Should this be optional via the `query? :string`

Comment (corivera) [reply src/main.ts]: It gets set as undefined if not needed.

**Comment by corivera** [comment]
Closing this PR temporarily until the other EditData PRs are ready to go.

**Comment by corivera** [comment]
@anthonydresser Reopened this PR. Can you take another look at it?

**Pull request merged**